### ISSUE TITLE
[Bug] Sorting with multiple tags

### ISSUE THREAD

**Issue by 0xcfff** [benrhughes/todotxt.net#415]
Suppose there is a task with multiple projects specified on it. When tasks are grouped by project and such a task is selected and any change to this task occurs, the selection works randomly and sometimes jumps to another project with the same task, not the one where the selection was before the change. That might also be the case for multiple contexts specified.

**Comment by remingtonsmith** [comment]
Yes, I sometimes tag a task with more than one project or use project tags and also use project tags much like context tags so that I can easily reference lists of contexts without having to do anything different than leave the view on sort by project.  The unpredictability of sorting when using multiple project tags is a problem.

**Comment by remingtonsmith** [comment]
This is not a full solution, but it might be somewhat helpful pending a software fix.  I've noticed that the sorting malfunction appears to affect only the project tags after the first one, such that, if I change the order of the project tags to where my "primary" project tag is the first one listed, the sorting of the most important project in Sort by Project view functions as expected.
